pkg/apiserver/apiserver.go

```diff
@@ -58,5 +61,22 @@ func (c CompletedConfig) New() (*ServiceCatalogAPIServer, error) {
 		GenericAPIServer: genericServer,
 	}
 
+	glog.Infoln("make and install the apis")
+
+	apiGroupInfo := genericapiserver.NewDefaultAPIGroupInfo(servicecatalog.GroupName)
+	// giving it v1alpha1 version
+	apiGroupInfo.GroupMeta.GroupVersion = v1alpha1.SchemeGroupVersion
+
+	// TODO make storage work
+	//v1alpha1storage := map[string]rest.Storage{}
```

Comment:
work being done in https://github.com/kubernetes-incubator/service-catalog/pull/172